tests/cypress/integration/actions_objects2/case_108_rotated_bounding_boxes.js

```diff
@@ -0,0 +1,129 @@
+    const createRectangleTrack2Points = {
+        points: 'By 2 Points',
+        type: 'Track',
+        labelName,
+        firstX: createRectangleShape2Points.firstX,
+        firstY: createRectangleShape2Points.firstY - 150,
+        secondX: createRectangleShape2Points.secondX,
+        secondY: createRectangleShape2Points.secondY - 150,
+    };
+
+    const transformMatrixShape = [];
+
+    function testShapeRotate(shape, x, y, expectedRorate, shift) {
+        cy.get(shape)
+            .trigger('mousemove')
+            .trigger('mouseover')
+            .should('have.class', 'cvat_canvas_shape_activated');
+        cy.get('.svg_select_points_rot')
+            .should('be.visible')
+            .and('have.length', 1)
+            .trigger('mousemove')
+            .trigger('mouseover')
+            .trigger('mousedown', { button: 0 });
+        if (shift) {
+            cy.get('.cvat-canvas-container').trigger('mousemove', x, y, { shiftKey: true });
+        } else {
+            cy.get('.cvat-canvas-container').trigger('mousemove', x, y);
+        }
+        cy.get('#cvat_canvas_text_content').should('contain.text', expectedRorate);
+        cy.get('.cvat-canvas-container').trigger('mouseup');
+        cy.get(shape).should('have.attr', 'transform');
+    }
+
+    before(() => {
+        cy.openTask(taskName);
+        cy.openJob();
+        Cypress.config('scrollBehavior', false);
+        cy.createRectangle(createRectangleShape2Points);
+        cy.createRectangle(createRectangleTrack2Points);
+    });
+
+    describe(`Testing case "${caseId}"`, () => {
+        it('Check that bounding boxes can be rotated.', () => {
+            testShapeRotate('#cvat_canvas_shape_1', 350, 150, '11.4°');
+            testShapeRotate('#cvat_canvas_shape_2', 350, 150, '26.6°');
+            cy.get('#cvat_canvas_shape_2').then((shape2) => {
+                const shapeAttrs = shape2.attr('transform').split('(')[1].split(')')[0].split(',');
+                for (const i of shapeAttrs) {
+                    transformMatrixShape.push(Number(i).toFixed(3));
+                }
+            });
+        });
+
+        it('Check interpolation, merging/splitting rotated shapes.', () => {
+            // Check track roration on all frames
+            for (let frame = 1; frame < 10; frame++) {
+                cy.goToNextFrame(frame);
+                cy.get('#cvat_canvas_shape_2').should('have.attr', 'transform').then(($transform) => {
+                    const transform = $transform.split('(')[1].split(')')[0].split(',');
+                    const transformMatrix = [];
+                    for (const i of transform) {
+                        transformMatrix.push(Number(i).toFixed(3));
+                    }
+                    expect(transformMatrix).to.deep.eq(transformMatrixShape);
+                });
+            }
+            // Merge tracks
+            cy.get('.cvat-merge-control').click();
+            cy.get('#cvat_canvas_shape_2').click();
```

Comment:
You didn't merge anything here, just select one existing track, right?

Reply:
Sure. Thanks. Fixed.
Added merge shapes and verification by analogy with tracks.